website/grading.rst

```diff
@@ -99,27 +135,38 @@ will earn the highest grade for which you meet *all* requirements.
 +----+----------------+-------+-----+---+
 |    | Project Points | N+S+E | S+E | E |
 +====+================+=======+=====+===+
-| A  | 200+           | 11    | 10  | 8 |
+| A  | 300+           | 11    | 10  | 5 |
 +----+----------------+-------+-----+---+
-| A- | 150+           | 11    | 10  | 5 |
+| A- | 250+           | 11    | 10  | 3 |
 +----+----------------+-------+-----+---+
-| B+ | 100+           | 11    | 8   | 0 |
+| B+ | 200+           | 11    | 8   | 0 |
 +----+----------------+-------+-----+---+
-| B  | < 100          | 9     | 7   | 0 |
+| B  | < 200          | 9     | 7   | 0 |
 +----+                +-------+-----+---+
```

Comment:
add dashes here to complete row and get the alternating cells. 
Looks like on continuous cell in actual document: 
![image](https://github.com/uchicago-cs/cmsc22000/assets/86740254/772bb193-1ad3-4fb2-99c8-b6750b70a31f)